### ISSUE TITLE
Unsupported encoding. JSON must be UTF-8

### ISSUE THREAD

**Issue by Dadie** [egret85/ClassicNewTabPage#23]
Browser: Chromium 41.0.2272.76 (64-bit)
OS: Arch Linux (64-bit)
Locale:
```
  LANG=en_US.UTF-8
  LC_CTYPE="en_US.UTF-8"
  LC_NUMERIC="en_US.UTF-8"
  LC_TIME="en_US.UTF-8"
  LC_COLLATE="en_US.UTF-8"
  LC_MONETARY="en_US.UTF-8"
  LC_MESSAGES="en_US.UTF-8"
  LC_PAPER="en_US.UTF-8"
  LC_NAME="en_US.UTF-8"
  LC_ADDRESS="en_US.UTF-8"
  LC_TELEPHONE="en_US.UTF-8"
  LC_MEASUREMENT="en_US.UTF-8" 
  LC_IDENTIFICATION="en_US.UTF-8"
  LC_ALL=
```

I tried to install this extensions, but got the following error: 

```Package is invalid. Details: '/tmp/scoped_dir_ackB5o/CRX_INSTALL/_locales/is/messages.json: Line: 6, column: 34, Unsupported encoding. JSON must be UTF-8.'.```

Seems like some json-files are not valid UTF-8 files.

**Comment by egret85** [comment]
This has been fixed

**Comment by Dadie** [comment]
If this has been fixed, why do I still experience the same bug? 

The [locale is](https://github.com/egret85/ClassicNewTabPage/blob/master/src/_locales/is/messages.json),[locale gl](https://github.com/egret85/ClassicNewTabPage/blob/master/src/_locales/gl/messages.json) and [locale cy](https://github.com/egret85/ClassicNewTabPage/blob/master/src/_locales/cy/messages.json) files are not valid encoded in UTF-8. 

And that has not been fixed.

**Comment by egret85** [comment]
I did not get errors with either of those files.  I was getting one with hy locale that I fixed.  I just removed and reinstalled v1.1.4 from the store that I published yesterday and got no errors.

**Comment by egret85** [comment]
I'm on Windows though so could be a difference between OS.  If you get it fixed, send a pull request.

**Comment by egret85** [comment]
Fixed in pull request #24
